counterpoll/main.py

```diff
@@ -81,6 +83,46 @@ def disable():
     port_info['FLEX_COUNTER_STATUS'] = 'disable'
     configdb.mod_entry("FLEX_COUNTER_TABLE", "PORT", port_info)
 
+# Port buffer drop counter commands
+@cli.group()
+def port_buffer_drop():
+    """ Port buffer drop  counter commands """
+
+@port_buffer_drop.command()
+@click.argument('poll_interval', type=click.IntRange(30000, 300000))
+def interval(poll_interval):
+    """
+    Set port_buffer_drop counter query interval
+    This counter group causes high CPU usage when polled,
+    hence the allowed interval is between 30s and 300s.
+    This is a ahort term solution and
+    should be changed once the performance is enhanced
+    """
+    configdb = swsssdk.ConfigDBConnector()
+    configdb.connect()
+    port_info = {}
+    if poll_interval is not None:
```

Comment:
```suggestion
    if poll_interval:
```
Since `poll_interval` can never == 0 can we do it this way?